front-end/src/components/comments/EditComment.tsx

```diff
@@ -58,18 +49,17 @@ const EditComment = () => {
     const onSubmit = async (comment: React.FormEvent<HTMLFormElement>) => {
           
         comment.preventDefault();
-        await isTokenExpired();
         if (body.length > 0){bodyDef = body}
         const payload = {body: bodyDef, post_id: params.post_id, user_id: decodeUserToken.user_id, token: user_token};
         const headers = {'Authorization': 'Bearer '+user_token}
         const response: any = await client.put(baseURL+`/posts/${params.post_id}/comments/${params.id}/edit`,payload,{headers: headers});
```

Comment:
Why are we using any. it is not recommend to use any in typescript. Because if we keep using any in our ts file. that specific file is prone to error and it doesnt any make sense if we are going to use typescript.